app/styles/calendar.css

```diff
@@ -1,3 +1,4 @@
+/* stylelint-disable no-descending-specificity */
```

Comment:
I don't think we should disable the linter here unless I'm missing something 🤔

Reply:
Thanks for the feedback, I had a second look & looked up a CSS Selectors reference. Turns out all I needed to do was remove the `>` symbol and my fix still functions as intended. There is a very subtle difference between `.classname > .classname` and `.classname .classname`

Reply:
@Sendouc nevermind, linter still fails :(

https://github.com/Sendouc/sendou.ink/actions/runs/3451373555/jobs/5760549697

I actually do not know how to fix this unless we disable the linter rule. It's an odd CSS Selector issue where if I rearrange the order like the linter asks me to, my fix ends up not working at all & not fixing the originally reported issues.

Reply:
@Sendouc I had another look today & found out I didn't have to disable the rule after all!

All I have to do is make sure that the CSS class with lower specificity (`.calendar__week__relative`) is defined earlier in the .css file than everywhere else that it is used as a higher specificity (i.e., deeper level) CSS property.

Reference: https://stackoverflow.com/a/57871548

So it all works out now & it should be good to be merged whenever you're ready. I double-checked that my fix still works & indeed it does!